### DISCUSSION TITLE
SPM 25.01 release candidate

### DISCUSSION THREAD

**Discussion by SPMcentral** [spm/spm#45] [Announcements]
I am delighted to inform you that we are preparing a major new version of SPM; the first since SPM12 a decade ago.  

SPM 25.01 will be released on **6th January 2025** via Github. Going forward, we will issue one major release per year. 

### Testing
Please test the release candidate, [SPM 25.01.rc3.zip](https://github.com/spm/spm/releases/download/25.01.rc3/spm_25.01.rc3.zip), against your code or toolbox as appropriate. There is also [Standalone SPM](https://github.com/spm/spm/releases/tag/25.01.rc3), which doesn’t require a MATLAB license. If you find any bugs, please report them using our [Issues page on Github](https://github.com/spm/spm/issues). To discuss the release, please use the dedicated thread on the [Github discussion forum](https://github.com/spm/spm/discussions/44).

### SPM versions
A particularly important change for toolbox developers is the update to the SPM version number, which could impact automated checks for SPM. For more information, please see the [documentation on SPM version numbers](https://www.fil.ion.ucl.ac.uk/spm/docs/development/versioning/). Updated Docker and Singularity containers, as well as a new Python interface to Standalone SPM, will be issued shortly after the release.

Kind regards
Peter Zeidman
On behalf of the SPM Dev Team

**Comment by ChristophePhillips** [comment]
[like]  Phillips Christophe reacted to your message:
________________________________
From: Statistical Parametric Mapping ***@***.***>
Sent: Tuesday, November 19, 2024 10:14:53 AM
To: spm/spm ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [spm/spm] SPM 25.01 release candidate (Discussion #45)


I am delighted to inform you that we are preparing a major new version of SPM; the first since SPM12 a decade ago.

SPM 25.01 will be released on 6th January 2025 via Github. Going forward, we will issue one major release per year.

Testing

Please test the release candidate, SPM 25.01.rc3.zip<https://github.com/spm/spm/releases/download/25.01.rc3/spm_25.01.rc3.zip>, against your code or toolbox as appropriate. There is also Standalone SPM<https://github.com/spm/spm/releases/tag/25.01.rc3>, which doesn’t require a MATLAB license. If you find any bugs, please report them using our Issues page on Github<https://github.com/spm/spm/issues>. To discuss the release, please use the dedicated thread on the Github discussion forum<https://github.com/spm/spm/discussions/44>.

SPM versions

A particularly important change for toolbox developers is the update to the SPM version number, which could impact automated checks for SPM. For more information, please see the documentation on SPM version numbers<https://www.fil.ion.ucl.ac.uk/spm/docs/development/versioning/>. Updated Docker and Singularity containers, as well as a new Python interface to Standalone SPM, will be issued shortly after the release.

Kind regards
Peter Zeidman
On behalf of the SPM Dev Team
